lang/syn/src/lib.rs

```diff
@@ -661,9 +662,19 @@ pub enum InitKind {
     Program { owner: Option<Expr> },
     // Owner for token and mint represents the authority. Not to be confused
     // with the owner of the AccountInfo.
-    Token { owner: Expr, mint: Expr },
-    AssociatedToken { owner: Expr, mint: Expr },
-    Mint { owner: Expr, decimals: Expr },
+    Token {
+        owner: Expr,
+        mint: Expr,
+    },
+    AssociatedToken {
+        owner: Expr,
+        mint: Expr,
+    },
+    Mint {
+        owner: Expr,
+        freeze_authority: Option<Expr>,
+        decimals: Expr,
+    },
```

Comment:
Not sure why my rust fmt formats this different :/

Reply:
you mean the order of the fields? or just the indentation

Reply:
Yeah just indentation stuff—it added newlines to `Token` and `AssociatedToken` that I wasn't expecting (my change only touches `Mint`).